### DISCUSSION TITLE
能否添加使B站网页端获取App一样的首页推荐视频

### DISCUSSION THREAD

**Discussion by hug1013** [the1812/Bilibili-Evolved#4294] [功能建议 / Ideas]
因为网页端的推荐视频不如手机app推荐的视频更加符合日常观看以及喜好
已有脚本实现:https://github.com/magicdawn/bilibili-app-recommend
但是想避免和Bilibili Evolved会有冲突所以请问下能否添加这个功能 并且优化？

**Comment by Geeyun-JY3** [comment]
#1203、#2913
